src/components/MovieTile/MovieTile.js

```diff
@@ -2,12 +2,14 @@ import React from 'react'
 import {Link} from 'react-router-dom'
 
 const MovieTile = (props) => {
+  const foundRating = props.userMovieRating.find(rating => rating.movie_id === props.movie.id)
   return (
     <article className='movie-tile'>
       <Link to={`/MovieShowPage/${props.movie.id}`}>
         <img src={props.movie.poster_path} alt={props.movie.title + " poster"}/>
       </Link>
-      <h4 className='movie-title'>Rating: {props.movie.average_rating}</h4>
+      <h4 className='movie-rating'>Rating: {props.movie.average_rating.toFixed(1)}</h4>
+      { (foundRating) ? <h4 className='movie-user-rating'>Your Rating: {foundRating.rating}</h4> : '' }
```

Comment:
Nice job with this conditional render! What's a more informative bit of text we could give the user if they don't have a rating for this movie yet?

Reply:
Likely we could say something more informative with a verb like "leave your rating here".